docs/build/apps/wallet/sep7.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: URI Scheme to facilitate delegated signing
+sidebar_position: 80
+---
+
+import Header from "./component/header.mdx";
+import { WalletCodeExample as CodeExample } from "@site/src/components/WalletCodeExample";
+
+<Header />
+
+The [Sep-7] standard defines a way for a non-wallet application to construct a URI scheme that represents a specific transaction for an account to sign. The scheme used is `web+stellar`, followed by a colon. Example: `web+stellar:<operation>?<param1>=<value1>&<param2>=<value2>`
+
+## Tx Operation
+
+The tx operation represents a request to sign a specific transaction envelope, with [some configurable parameters].
+
+<CodeExample>
+
+```typescript
+const sourceAccountKeyPair = "G...";
+const destinationAccountKeyPair = "G...";
+const txBuilder = await stellar.transaction({
+  sourceAddress: sourceAccountKeyPair,
+});
+const tx = txBuilder.createAccount(destinationAccountKeyPair).build();
+const xdr = encodeURIComponent(tx.toEnvelope().toXDR().toString("base64"));
+const callback = encodeURIComponent("https://example.com/callback");
+const txUri = `web+stellar:tx?xdr=${xdr}&callback=${callback}`;
+const uri = wallet.parseSep7Uri(txUri);
+// uri can be parsed and transaction can be signed/submitted by an application that implements Sep-7
+```
+
+</CodeExample>
+
+You can set replacements to be made in the xdr for specific fields by the application, these will be added in the [Sep-11 transaction representation format] to the URI.
+
+<CodeExample>
+
+```typescript
+const uri = new Sep7Tx(txUri);
+uri.addReplacement({
+  id: "X",
+  path: "sourceAccount",
+  hint: "account from where you want to pay fees",
+});
+```
+
+</CodeExample>
+
+You can assign parameters after creating the initial instance using the appropriate setter for the parameter.
+
+<CodeExample>
+
+```typescript
+const sourceAccountKeyPair = "G...";
+const destinationAccountKeyPair = "G...";
+const txBuilder = await stellar.transaction({
+  sourceAddress: sourceAccountKeyPair,
+});
+const tx = txBuilder.createAccount(destinationAccountKeyPair).build();
+const uri = wallet.Sep7Tx.forTransaction(tx);
+uri.callback = "https://example.com/callback";
+uri.msg = "here goes a message";
+uri.toString(); // encodes everything and converts to a uri string
+```
+
+</CodeExample>
+
+## Pay Operation
+
+The pay operation represents a request to pay a specific address with a specific asset, regardless of the source asset used by the payer. You can [configure parameters] to build the payment operation.
+
+<CodeExample>
+
+```typescript
+const destination = "G...";
+const assetIssuer = "G...";
+const assetCode = "XLM";
```

Comment:
If we are setting an `assetIssuer` we should set a different asset code here since `XLM` does not have an issuer. I'd suggest using "USDC" instead.

Reply:
Thanks, makes sense. Updated in ff3c968ffdca003c792112ead948897e000d68e7